CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+# Contributing
+
+Your contributions are always appreciated! To make it easier for both you and the maintainers to resolve your issues or ship your changes, please refer to the guidance below.
+
+- [Bug Reports](#bug-reports)
+- [Feature Requests](#feature-requests)
+- [Pull Requests](#pull-requests)
+
+## Bug Reports
+
+If you have ran into an issue using CreateAPI, when creating an issue on GitHub, please make sure that you include the following information:
+
+- The version of CreateAPI that you were using
+- A reproducible example including the schema, the config and the options that you passed to the generate command
+
+When the instructions are clear and its easy for a maintainer to reproduce the issue, it makes it easier for everybody to find a solution.
+
+While its not required, if you can reproduce the issue in the existing tests  and open a draft Pull Request that demonstrates the issue, that is even better! You don't have to know how to fix it, but its a great start.
+
+## Feature Requests
+
+We want CreateAPI to be full of rich and useful features. If you have an idea, please open an issue to start a discussion!
+
+## Pull Requests
+
+Submitting Pull Requests is always welcome, this could be to fix bugs, refactor code, implement new features or even to add or update documentation.
+
+If you are changing code, please be sure to make sure that the tests pass locally before submitting your Pull Request. You can do this by running `swift test` in terminal (or <kbd>⌘</kbd> + <kbd>U</kbd> in Xcode).
+
+### Test Fixtures
+
+The CreateAPITests target has a series of expected fixtures committed to the repository and when running tests, to verify that the generator works as expected, the output generated by the test must match the contents committed to the repo.
+
+If you are making a change, ether to fix/reproduce a bug, or to add a new feature, you can enable some helpers in **Snapshots.swift** to help you:
```

Comment:
Pedantic change, but I think files would be in `code` because **bold** can be hard to see.

Reply:
heh yeah, I guess it depends a bit on the importance but I follow the Ray Wenderlich formatting guidelines (they're not public I don't think, but I occasionally tech edit there) when I use bolding for filenames. 

I'll stick with using bold because I don't think there is too much importance on the filenames in here 🙇